src/main/webapp/partials/instructorHelpQuestions.jsp

```diff
@@ -358,589 +358,1614 @@
-                            <span class="input-group-addon">
-                              <input type="radio" class="disabled_radio" disabled="">
-                            </span>
-                                <input class="form-control" type="text" disabled="" name="mcqOption-0" id="mcqOption-0-6" value="Yes">
-                                <span class="input-group-btn">
-                              <button class="btn btn-default removeOptionLink" type="button" id="mcqRemoveOptionLink" onclick="removeMcqOption(0,6)" style="display:none" tabindex="-1" disabled="">
-                                <span class="glyphicon glyphicon-remove">
-                                </span>
-                              </button>
-                            </span>
-                              </div>
-                            </div>
-                            <div class="margin-bottom-7px" id="mcqOptionRow-1-6">
-                              <div class="input-group width-100-pc">
-                            <span class="input-group-addon">
-                              <input type="radio" class="disabled_radio" disabled="">
-                            </span>
-                                <input class="form-control" type="text" disabled="" name="mcqOption-1" id="mcqOption-1-6" value="No">
-                                <span class="input-group-btn">
-                              <button class="btn btn-default removeOptionLink" type="button" id="mcqRemoveOptionLink" onclick="removeMcqOption(1,6)" style="display:none" tabindex="-1" disabled="">
-                                <span class="glyphicon glyphicon-remove">
-                                </span>
-                              </button>
-                            </span>
-                              </div>
-                            </div>
+              <div class="col-sm-12 margin-bottom-7px">
+                <div class="col-sm-4 padding-0">
+                  <input type="checkbox" class="nonDestructive" id="mcqAssignWeights-1" name="mcqAssignWeights" checked="" disabled="">
+                  <span data-toggle="tooltip" data-placement="top" data-original-title="Assign weights to the choices for calculating statistics." class="tool-tip-decorate"> Choices are weighted </span>
+                </div>
+                <div class="col-sm-2 padding-left-45px align-center">
+                  Weights <span class="glyphicon glyphicon-arrow-down"></span>
+                </div>
+              </div>
+              <br>
+              <div class="col-sm-6">
+                <div id="mcqChoiceTable-1">
+                  <div class="row">
+                    <div class="col-sm-9" id="mcqChoices-1">
+                      <div class="input-group margin-bottom-7px" id="mcqOptionRow-0-1">
+              <span class="input-group-addon">
+                <input type="radio" class="disabled_radio" disabled="">
```

Comment:
Can you change the class to dash-cased?

Reply:
@bqnguyen94 Do you mean like `disabled-radio`? But the original class name is `disabled_radio` only? Don't you think not having the original class name might create confusion afterwards?